pkg/definition/definition.go

```diff
@@ -284,15 +284,23 @@ func (def *Definition) FromCUE(val *cue.Value, templateString string) error {
 					return err
 				}
 				for _k, _v := range _annotations {
-					annotations[UserPrefix+_k] = _v
+					if strings.Contains(_k, "oam.dev") {
```

Comment:
So it means we allow user to edit system generated annotations? Like addon.oam.dev?

Reply:
it's not about edit, it's generated without redundant prefix